ios/Plugin/FirebaseAuthentication.swift

```diff
@@ -89,6 +91,34 @@ public typealias AuthStateChangedObserver = () -> Void
         self.oAuthProviderHandler?.signIn(call: call, providerId: "yahoo.com")
     }
 
+    @objc func signInWithCustomToken(_ call: CAPPluginCall) {
+        if config.skipNativeAuth == true {
+            call.unimplemented(self.errorCustomTokenUnimplemented)
+            return
+        }
+
+        let token = call.getString("token")
+        if token == nil {
+            call.reject(self.errorInvalidCustomToken, nil)
+            return
+        }
```

Comment:
```suggestion
        let token = call.getString("token", "")
```


I would keep the code as simple as possible at this point. 
Extensive error handling will be added soon.